ui/package.json

```diff
@@ -26,18 +26,26 @@
     "@fortawesome/free-brands-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.16",
+    "@monaco-editor/react": "^4.3.1",
     "@mui/icons-material": "^5.0.0",
     "@mui/material": "^5.0.0",
     "@mui/styles": "^5.0.0",
     "@reduxjs/toolkit": "^1.6.1",
+    "ace-builds": "^1.4.13",
+    "brace": "^0.11.1",
```

Comment:
do we need this ?

Reply:
Ace-builds is needed. I'll remove the` brace` dependency.